packages/@react-aria/dnd/src/DragManager.ts

```diff
@@ -70,6 +70,13 @@ export function beginDragging(target: DragTarget, formatMessage: (key: string) =
   requestAnimationFrame(() => {
     dragSession.setup();
 
+    if (getDragModality() === 'virtual') {
+      let {element} = target;
+      element.blur();
+      setTimeout(() => element.focus(), 100);
```

Comment:
why?

Reply:
In my testing, simply blurring and refocusing immediately, or even waiting up to 85ms, was not enough to force NVDA or JAWS into forms mode. 100ms seems to provide enough of a delay so that the screen reader picks up on the ancestor `role="application"`. Forcing forms mode would work better if we moved focus to the first drop target using `dragSession.next();`, as we do when `getDragModality() === 'keyboard'`, but I think we want the messaging that accompanies `getDragModality() === 'keyboard'` without shifting focus to a drop target.